### PR TITLE
Add renderValue and getFieldSetsFields utilities

### PR DESCRIPTION
**Pull request by rkuykendall**
```ts
renderValue (fieldConfigPartial: IFieldConfigPartial, model?: IModel): React.ReactNode
getFieldSetsFields (fieldSets: IFieldSet[]): IFieldConfig[]
```

### CODE REVIEW

File: test/utilities/FormManager.test.tsx [hunk 1]
```diff
@@ -54,4 +56,33 @@ describe('FormManager', () => {
...
+  });
+
+  it(`Handles 500 responses`, async () => {
+    const fieldSets = [[{ field: 'name' }]]
+      , name = faker.name.firstName()
+      , id = faker.random.uuid()
+      , notId = faker.random.uuid()
+      , response = {
+        config: {},
+        data: '<!DOCTYPE html><html lang="en"><head /></html>',
```
Comment (SHA-4) [inline test/utilities/FormManager.test.tsx]: Why include all this extra response information when it's only checking status?

File: test/utilities/FormManager.test.tsx [hunk 2]
```diff
@@ -54,4 +56,33 @@ describe('FormManager', () => {
...
+    const formManager = await getFormManager(fieldSets, { id, name, notId });
+    expect(formManager.formModel).toEqual({ name, id });
+  });
+
+  it(`Handles 500 responses`, async () => {
+    const fieldSets = [[{ field: 'name' }]]
+      , name = faker.name.firstName()
+      , id = faker.random.uuid()
+      , notId = faker.random.uuid()
+      , response = {
```
Comment (SHA-4) [inline test/utilities/FormManager.test.tsx]: I would rename this `badResponse`

File: src/utilities/FormManager.ts [hunk 3]
```diff
@@ -144,8 +144,9 @@ class FormManager {
   }
 
   private handleBackendResponse (response?: any) {
+    // console.log('validateThenSaveCallback', response);
```
Comment (SHA-4) [inline src/utilities/FormManager.ts]: Leftover comment

File: test/utilities/FormManager.test.tsx (hunk 1 above)
Comment (rkuykendall) [reply test/utilities/FormManager.test.tsx]: I copied it directly from a real response. This string here is unexpected by the code so I wanted it to be there. I realize it _shouldn't_ affect anything but hey that's what tests are for, right?

File: test/utilities/FormManager.test.tsx [hunk 4]
```diff
@@ -57,22 +57,12 @@ describe('FormManager', () => {
     expect(formManager.formModel).toEqual({ name, id });
   });
 
-  it(`Correctly maintains id`, async () => {
```
Comment (rkuykendall) [inline test/utilities/FormManager.test.tsx]: Duplicate

**Pull request merged**